buffer/src/main/java/io/netty/buffer/AbstractByteBuf.java

```diff
@@ -33,6 +33,7 @@
 import java.nio.channels.ScatteringByteChannel;
 import java.nio.charset.Charset;
 
+import static io.netty.util.internal.MathUtil.isOutOfBounds;
```

Comment:
I'm not sure about MathUtils either, but it needs to go in the `common` module because it is used by `AsciiString` (which lives in the common module).  Maybe I should create an `ArrayUtils` in `common`?